e2e-polybft/e2e/bridge_test.go

```diff
@@ -183,16 +184,37 @@ func TestE2E_Bridge_Transfers(t *testing.T) {
 
 		currentEpoch := currentExtra.Checkpoint.EpochNumber
 
-		require.NoError(t, waitForRootchainEpoch(currentEpoch, 3*time.Minute,
-			rootchainTxRelayer, polybftCfg.Bridge.CheckpointManagerAddr))
-
 		exitHelper := polybftCfg.Bridge.ExitHelperAddr
 		childJSONRPC := validatorSrv.JSONRPCAddr()
 
-		for exitEventID := uint64(1); exitEventID <= transfersCount; exitEventID++ {
-			// send exit transaction to exit helper
-			err = cluster.Bridge.SendExitTransaction(exitHelper, exitEventID, childJSONRPC)
-			require.NoError(t, err)
+		successfullExitTransactions := make([]bool, transfersCount)
+
+		for i := 0; i < numberOfAttempts; i++ {
+			require.NoError(t, waitForRootchainEpoch(currentEpoch+uint64(i), 3*time.Minute,
+				rootchainTxRelayer, polybftCfg.Bridge.CheckpointManagerAddr))
+
+			for exitEventID := uint64(1); exitEventID <= transfersCount; exitEventID++ {
+				// send exit transaction to exit helper
+				if !successfullExitTransactions[exitEventID-1] {
```

Comment:
I would change this to make it more clear:

```suggestion
				if successfullExitTransactions[exitEventID-1] {
				    continue
				}
				
				err = cluster.Bridge.SendExitTransaction(exitHelper, exitEventID, childJSONRPC)
```